product-mini/platforms/linux-sgx/enclave-sample/Enclave/Enclave.cpp

```diff
@@ -279,6 +316,34 @@ handle_cmd_unload_module(uint64 *args, uint32 argc)
     LOG_VERBOSE("Unload module success.\n");
 }
 
+#if WASM_ENABLE_LIB_RATS != 0
+extern "C" {
+char *
+wasm_runtime_get_module_hash(wasm_module_t module);
+}
```

Comment:
Move to wasm_export.h, since the function is implemented in this cpp file, if it isn't declared in .h file with `extern "C" { .. }`, it will be compiled as a c++  API, and eventually link error occurs.